frontend/tests/console/topics/topic-messages-production.spec.ts

```diff
@@ -0,0 +1,254 @@
+// spec: specs/topics.md
+// seed: tests/seed.spec.ts
+
+import { expect, test } from '@playwright/test';
+
+import { TopicPage } from '../utils/TopicPage';
+
+test.use({
+  permissions: ['clipboard-write', 'clipboard-read'],
+});
```

Comment:
We should also make sure we only run this test on Chromium browsers since otherwise the permissions (WebKit/Firefox) would break the test.